pkg/util/refreshable/refreshable.go

```diff
@@ -63,13 +64,16 @@ func (a *authorizer) RefreshWithContext(ctx context.Context, log *logrus.Entry)
 			azureerrors.IsInvalidSecretError(err) {
 			return false, nil
 		}
-
 		return false, err
 	}
 
 	return true, nil
 }
 
+func (a *authorizer) RefreshWithContextRawError(ctx context.Context, log *logrus.Entry) error {
```

Comment:
Why do we need `log` here?

Reply:
yes log can be ignored, as we are not logging anything for the new function